homework/src/nyc/c4q/ahhhlvin/Animal.java

```diff
@@ -0,0 +1,35 @@
+package nyc.c4q.ahhhlvin;
+
+import java.util.List;
+
+
+
+    public class Animal implements Comparable<Animal> {
+
+        private String speciesname;
```

Comment:
I would recommend calling it `speciesName` so that it follows java's naming convention. Nice job making it private!